context-free.md

```diff
@@ -2,21 +2,61 @@
 
 ##  Who is this programming language for?
 
+Context Free is for artists iterested in exploring generative art.
 
 ## What is easy to do in this language? Why is it easy?
 
+It's easy to generate art that has a lot of symmetry. One of the easiest tricks
+for generating an image is to create recursive shape rules, and this tends to
+lead to shapes with a recursive structure, which results in symmetry. The
+handful of tree-like example programs confirm this.
 
 ## What is hard to do in this language? Why is it hard?
 
+It would be hard to replicate a picture of some real object in the language.
```

Comment:
I also agree with the challenges of creating a "real object" using this language -- ContextFree's strength is its ability to create objects with symmetry, but few objects in the real world are perfectly symmetrical.